scripts/run_backend_tests.sh

```diff
@@ -75,6 +80,9 @@ source $(dirname $0)/setup_gae.sh || exit 1
 # Install third party dependencies
 bash scripts/install_third_party.sh
 
+SKIP_COMPILATION=False
```

Comment:
Let's make this true by default since most tests don't require compilation, right?

Reply:
If we name it `DO_COMPILATION`, it should be false by default so that when user is using the flag we set it to true and then do the compilation, right?

Reply:
I have updated the PR by renaming the flag and changing the default according to my previous comment. Please do let me know if I misunderstood your query and I will make updates according to that then.